variables.tf

```diff
@@ -81,7 +89,7 @@ variable "lambda_subnet_bits" {
 
 variable "map_public_ip_on_launch" {
   type        = bool
-  default     = false
+  default     = true
```

Comment:
Let's not change this default

Reply:
sure, in testing on tag 1.7.0 it appeared to look to overwrite based on our code as does not appear set

Reply:
will add this var to our local modules